src/components/accordion-item/accordion-item.tsx

```diff
@@ -170,7 +169,6 @@ export class AccordionItem {
           </div>
           <div
             aria-expanded={toAriaBoolean(this.expanded || this.active)}
```

Comment:
Do we still need this `aria-expanded` here?

Reply:
As well as the other aria attributes and roles here?

Reply:
Originally it was needed, but these changes might have mitigated the issue. Updating to exclude the `aria-expanded` and `role`.